src/main/java/com/google/sps/data/Constants.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.data;
+
+public final class Constants {
+    private Constants() {}
```

Comment:
nit: move the constructor the the bottom of the class since its private and does nothing.